tools/flang2/flang2exe/dtypeutl.cpp

```diff
@@ -512,7 +512,10 @@ alignment(DTYPE dtype)
   case TY_LOG128:
   case TY_FLOAT128:
   case TY_CMPLX128:
-    return dtypeinfo[ty].align;
+    align_bits = dtypeinfo[ty].align;
```

Comment:
`align_bits` is a misleading variable name. Can we rename it to `align_bytes`?

Reply:
> `align_bits` is a misleading variable name. Can we rename it to `align_bytes`?

Yes, you are right. I have refactored this part of the code.